src/translations/fr.json

```diff
@@ -23,5 +23,12 @@
     "content2": "Soyez le premier à marquer 21 points dans cette bataille explosive d'intelligence. Montez une offense vaillante tout en perturbant votre adversaire avec des astuces perfides. Avez-vous ce qu'il faut pour devenir le Seigneur des Profondeurs ?",
     "readRules": "Lire les règles",
     "joinDiscord": "Rejoignez notre communauté sur Discord"
+  },
+  "lobby":{
+    "lobbyFor": "Faire pression pour",
+    "ready": "prêt",
```

Comment:
Issue: Value for 'ready' is all caps in `en.js` Keep these casing consistent between locales.

Reply:
Good call!